src/DeploymentManager/DeploymentManager/help/Get-AzDeploymentManagerArtifactSource.md

```diff
@@ -0,0 +1,138 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.DeploymentManager.dll-Help.xml
+Module Name: Az.DeploymentManager
+online version: https://docs.microsoft.com/en-us/powershell/module/az.deploymentmanager/get-azdeploymentmanagerartifactsource
+schema: 2.0.0
+---
+
+# Get-AzDeploymentManagerArtifactSource
+
+## SYNOPSIS
+
+Gets the Artifact source.
+## SYNTAX
+
+### Interactive (Default)
+```
+Get-AzDeploymentManagerArtifactSource [-ResourceGroupName] <String> [-Name] <String>
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ResourceId
+```
+Get-AzDeploymentManagerArtifactSource [-ResourceId] <String> [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+### InputObject
+```
+Get-AzDeploymentManagerArtifactSource [-ArtifactSource] <PSArtifactSource>
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+{{Fill in the Description}}
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> {{ Add example code here }}
```

Comment:
@deveshguha please make sure all of your cmdlet markdown files have examples with corresponding output and descriptions

Reply:
@deveshguha the commands and descriptions were added for the examples, but it would be nice to have the corresponding output that users would see if they were to run the commands -- this can be done for the next iteration